.vscode/settings.json

```diff
@@ -28,5 +28,8 @@
       "args": ["-rcfile", "./.bashrc"]
     }
   },
-  "terminal.integrated.defaultProfile.linux": "bash"
+  "terminal.integrated.defaultProfile.linux": "bash",
+  "[javascriptreact]": {
```

Comment:
Please undo this formatter change.  We are using prettier, it's already in setting but you need to install it in vscode: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode